packages/backend-common/src/config.ts

```diff
@@ -185,6 +221,7 @@ export async function loadBackendConfig(options: {
   );
 
   config.setConfig(ConfigReader.fromConfigs(configs));
+  await updateRedactionMap(configs, options.logger);
```

Comment:
Perhaps we can split this into two steps to keep the redaction map handling sync? Where we load in the schema once async, and then the rest the the operations are sync.

Reply:
That sounds good :+1: I wasn't sure if the schemas could change during too?